### DISCUSSION TITLE
[FEATURE] Starter template for "combined" layout

### DISCUSSION THREAD

**Discussion by bastinald** [tabler/tabler#928] [Q&A] [answered]
Is there a blank/starter code available for the combined layout?

I could copy the code from the demo, but it comes with a ton of code I don't need.

Is there somewhere this is available I'm missing? Even the docs doesn't really tell me how to piece things together from scratch...

Thanks.

**Comment by rjd22** [comment, marked as answer]
@bastinald Normally I copy the code of the demo and clear out the stuff I don't need. You can also build your own demo with the scripts in this repo by cloning the repo and changing the layouts found in https://github.com/tabler/tabler/tree/dev/src/pages but I would say that is more work than just clearing out the layouts.

**Reply by robertandrews** [reply]
What "scripts" are these exactly?
I don't recognise this templating language. What's it for?

**Reply by dheineman** [reply]
The templating language is Liquid as used in Jekyll (see https://jekyllrb.com/docs/liquid/ ). The files/scripts in the pages folder
are used to build https://preview.tabler.io/ and during development as an easy way of having a full "site" that includes all elements of tabler which can be used to test your changes on.
